clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/Operation.java

```diff
@@ -17,10 +17,17 @@
 package org.ehcache.clustered.client.internal.store.operations;
 
 import org.ehcache.core.spi.function.Function;
+import org.ehcache.spi.serialization.Serializer;
 
-public interface Operation<K> extends Function<Operation<K>, Operation<K>> {
+import java.nio.ByteBuffer;
+
+public interface Operation<K, V> extends Function<Operation<K, V>, Operation<K, V>> {
 
   OperationCode getOpCode();
 
   K getKey();
+
+  V getValue();
```

Comment:
Given my comment on `PutIfAbsentOperation`, I am starting to be convinced that an `Operation` should not even have this method. We would introduce a different abstraction which is a `ResolvedOperation` which would then have a `getResult()` and be an extension of an operation ... which is a problem with regards to the OpCode ... or maybe not